### PR TITLE
refactor: Replace `fse.rename` with `fs.promises.rename`

### PR DESCRIPTION
**Pull request by sdas13**
Addresses: #8605

### CODE REVIEW

File: lib/utils/eventuallyUpdate.js [hunk 1]
```diff
@@ -6,6 +6,7 @@ const path = require('path');
 const os = require('os');
 const stream = require('stream');
 const { promisify } = require('util');
+const fs = require('fs');
```
Comment (pgrzesik) [inline lib/utils/eventuallyUpdate.js:9]: Let's require `fs.promises` as `fsp`:

```
const fsp = require('fs').promises;
```

**Review by pgrzesik — CHANGES_REQUESTED** [review]
Thanks a lot @sdas13 :bow: I only have one comment, please let me know what do you think :bow:

File: lib/utils/eventuallyUpdate.js (hunk 1 above)
Comment (sdas13) [reply lib/utils/eventuallyUpdate.js:9]: sure. makes sense

Comment (pgrzesik) [reply lib/utils/eventuallyUpdate.js:9]: Thanks, please request a review from me when it's ready for reviewing again :bow:

File: lib/plugins/standalone.js [hunk 2]
```diff
@@ -73,7 +73,7 @@ module.exports = class Standalone {
     }
     await fse.remove(BINARY_TMP_PATH);
     await pipeline(standaloneResponse.body, fs.createWriteStream(BINARY_TMP_PATH));
-    await fse.rename(BINARY_TMP_PATH, BINARY_PATH);
+    await fs.promises.rename(BINARY_TMP_PATH, BINARY_PATH);
```
Comment (pgrzesik) [inline lib/plugins/standalone.js:76]: Is there a reason as to why the same change was not applied here? (`fs.promises` -> `fsp`) ?

**Review by pgrzesik — CHANGES_REQUESTED** [review]
Hello @sdas13, thank you :bow: I have one question and we should be good to go

File: lib/plugins/standalone.js (hunk 2 above)
Comment (sdas13) [reply lib/plugins/standalone.js:76]: @pgrzesik In this file, `fs` was imported earlier and used as `fs.createWriteStream` (L75). I don't think we can use `fs.promises` with `createWriteStream`, so couldn't import `fs` as `fsp` for this case. Let me know if you have any other thoughts.

Comment (pgrzesik) [reply lib/plugins/standalone.js:76]: We can still import both `const fs = require('fs')` and in next line `const fsp = require('fs').promises`

**Review by pgrzesik — APPROVED** [review]
Thank you @sdas13, well done :clap:

**Pull request merged**